scripts/unicode/CaseFolding.txt

```diff
@@ -0,0 +1,1584 @@
+# CaseFolding-13.0.0.txt
```

Comment:
we can probably replace this file and the generated file with a dependency on the `unicode-13.0.0` package.